Disable MIPS VDSO until the cache issues have been sorted out.

Signed-off-by: Felix Fietkau <nbd@nbd.name>

--- a/arch/mips/vdso/Makefile
+++ b/arch/mips/vdso/Makefile
@@ -26,11 +26,11 @@ aflags-vdso := $(ccflags-vdso) \
 # the comments on that file.
 #
 ifndef CONFIG_CPU_MIPSR6
-  ifeq ($(call ld-ifversion, -lt, 22500000, y),y)
-    $(warning MIPS VDSO requires binutils >= 2.25)
+#  ifeq ($(call ld-ifversion, -lt, 22500000, y),y)
+#    $(warning MIPS VDSO requires binutils >= 2.25)
     obj-vdso-y := $(filter-out gettimeofday.o, $(obj-vdso-y))
     ccflags-vdso += -DDISABLE_MIPS_VDSO
-  endif
+#  endif
 endif
 
 # VDSO linker flags.
